src/main/support.js

```diff
@@ -6,10 +6,12 @@ const semver = {
   gte: require("semver/functions/gte"),
 };
 const arrayify = require("../utils/arrayify");
+// [prettierx] read info from package.json:
+const pkg = require("../../package.json");
 const coreOptions = require("./core-options").options;
 
-// [prettierx] quick workaround:
-const currentVersion = "2.2.0";
+// [prettierx] get info from package.json:
+const currentVersion = pkg["prettier-version"];
```

Comment:
@brodybits TODO:

- [ ] propose this kind of update in upstream Prettier code base, as this may help some other forks that want to keep their version < 1.0.0